pkgs/by-name/ba/bato/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  pkgs,
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bato";
+  version = "0.1.7";
+
+  src = fetchFromGitHub {
+    owner = "doums";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-i2gw8vXiKutq26ACzkVXH3kED7jAngSv2mNo9P3qXnA=";
+  };
+
+  cargoHash = "sha256-cqrSyZjfBSfF015nwOcRy2rkE4XzFPxJCQZGrqZXLEA=";
+
+  nativeBuildInputs = with pkgs; [
+    pkg-config
+    cmake
+  ];
+
+  buildInputs = with pkgs; [ libnotify ];
+
+  meta = with lib; {
+    description = "Small program to send battery notifications";
+    homepage = "https://github.com/doums/bato";
+    changelog = "https://github.com/doums/bato/releases/tag/v0.1.7";
+    license = with licenses; [ mpl20 ];
+    maintainers = [ ];
```

Comment:
Not going to add yourself as a maintainer here?

Reply:
I don't plan on maintaining it
But I can take care of it for 1-2 release cycles, I guess
shouldn't recieve much updates